modules/end-user-guide/examples/snip_che-editor-key-values.adoc

```diff
@@ -12,16 +12,24 @@
 | `eclipse/che-theia/next`
 | When using the `next` link:https://github.com/che-incubator/chectl/[chectl] channel, this IDE loads as a default without the URL parameter.
 
-| link:https://github.com/che-incubator/che-code[Che-Code]
+| link:https://github.com/che-incubator/che-code[Visual Studio Code]
```

Comment:
Can we really call it Visual Studio Code?

Suggestion based on the title in https://github.com/microsoft/vscode:

```suggestion
| link:https://github.com/che-incubator/che-code[Visual Studio Code - Open Source]
```